tools/mo/requirements_mxnet.txt

```diff
@@ -1,4 +1,4 @@
-mxnet~=1.2.0; sys_platform == 'win32'
+mxnet~=1.9.1; sys_platform == 'win32'
 mxnet>=1.7.0.post2,<=1.9.1; sys_platform != 'win32'
```

Comment:
It is better to combine these two lines into one:
```
mxnet>=1.7.0.post2,<=1.9.1
```
The same comment for networkx

Reply:
will you create a PR with your proposals?

Reply:
Yes, I will do.